features/page-breaks.yml

```
@@ -1,5 +1,54 @@
 name: Page breaks
 description: The `break-after`, `break-before`, `break-inside` CSS properties (along with `page-break-` aliases) control where printed pages start and end. Also known as pagination or page breaking.
-spec: https://drafts.csswg.org/css-break-3/#breaking-controls
+spec: https://drafts.csswg.org/css-break-4/#breaking-controls
 group: layout
 caniuse: css-page-break
+status:
+  compute_from: css.properties.page-break-after.avoid
```

Can we add this before merging?

```suggestion
status:
  # TODO: review this when https://github.com/web-platform-dx/web-features/issues/1878 is resolved
  compute_from: css.properties.page-break-after.avoid
```